sentry/src/main/java/io/sentry/SpanId.java

```diff
@@ -0,0 +1,20 @@
+package io.sentry;
+
+import java.util.UUID;
+
+public final class SpanId {
```

Comment:
can we use the `SentryId` for this class? they look exactly the same for me

Reply:
They have different length. We can't use SentryId here.

Reply:
oh just saw its, `substring(0, 16)`, true, thanks!